docs/notifications/events/components.md

```diff
@@ -0,0 +1,75 @@
+Component status updates emit the following events
+
+1. `component.status.healthy`
+2. `component.status.unhealthy`
+3. `component.status.warning`
+4. `component.status.error`
+5. `component.status.info`
+
+## Environment variables
+
+| Field       | Description                   | Schema                    | Optional |
+| ----------- | ----------------------------- | ------------------------- | -------- |
+| `component` | The component object          | [`Component`](#component) |          |
+| `agent`     | Agent details (if applicable) | [`Agent`](#check)         | `true`   |
+| `permalink` | A link to the health check    | `string`                  |          |
+
+### Component
+
+| Field                | Description                                          | Schema              | Optional |
+| -------------------- | ---------------------------------------------------- | ------------------- | -------- |
+| `id`                 | The id of the component                              | `uuid`              |          |
+| `topology_id`        | The id of the topology that generated this component | `uuid`              |          |
```

Comment:
I think we can remove agent_id - also the selectors, is_leaf, path etc - Basically anything that wouldn't make logical sense to filter on  or include in a message